setup.py

```diff
@@ -17,18 +17,21 @@ def run_tests(self):
         errno = pytest.main(self.test_args)
         sys.exit(errno)
 
+
 tests_require = ['pytest', 'pytest-flask', 'lxml', 'requests', 'cssselect']
 
+
 def data_file_names():
     for top in 'pyvocz/static', 'pyvocz/templates':
         for dirname, dirs, files in os.walk(top):
             for filename in files:
                 fullname = os.path.join(dirname, filename)
                 if filename not in dirs and fullname.endswith((
                         '.png', '.svg', '.jpg', '.css', '.html',
-                        )):
+                )):
                     yield os.path.relpath(fullname, 'pyvocz')
```

Comment:
I believe this complied to [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation), no need to change it.

Reply:
Still the indentation is visually more understandable (on first sight) than the "un-indentation".

Compare:
```
)):
    yield os.path.relpath(fullname, 'pyvocz')
```

To:
```
        )):
yield os.path.relpath(fullname, 'pyvocz')
```

Reply:
I disagree.

But again, I don't really care about this case. What I don't want is a checker that invents (and enforces) its own rules.